schema.json

```diff
@@ -39,6 +39,12 @@
         "type": "string",
         "pattern": "^(community|official)$"
       },
+      "generator": {
+        "type": "array",
+        "items": {
+          "pattern": "^((generator-.*)|data-service-generator)$"
```

Comment:
👍
we just need to remember to clean it up after we rename the data-service-generator